database_cleanup/tests/test_purge_fields.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2021 Camptocamp SA
+# Copyright 2024 360ERP (https://www.360erp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests.common import tagged
+
+from .common import Common, environment
+
+
+# Use post_install to get all models loaded more info: odoo/odoo#13458
+@tagged("post_install", "-at_install")
+class TestCleanupPurgeLineColumn(Common):
```

Comment:
Unrelated classname should be `TestCleanupPurgeFields`

Reply:
Right, thanks!